pkg/controller/syncset/syncset_controller.go

```diff
@@ -386,16 +387,28 @@ func findSyncSetInstanceForSelectorSyncSet(name string, syncSetInstances []*hive
 }
 
 func (r *ReconcileSyncSet) syncSetInstanceForSyncSet(cd *hivev1.ClusterDeployment, syncSet *hivev1.SyncSet) (*hivev1.SyncSetInstance, error) {
+	ssiLogger := r.logger.WithField("syncsetName", syncSet.Name)
 	cdRef := metav1.NewControllerRef(cd, hivev1.SchemeGroupVersion.WithKind("ClusterDeployment"))
 	hash, err := r.computeHash(syncSet.Spec)
 	if err != nil {
 		return nil, err
 	}
+
+	ssiLogger.Debug("Apply syncset labels to syncsetinstance")
```

Comment:
the debug stmt might be more useful if you say what label(s) you're talking about.
(same on lines 403, 434, 441)

Reply:
fixed.